.env.example

```
@@ -66,6 +66,19 @@ EXA_API_KEY=
+# ============================================================================
+# LLM SYNTHESIZER (OPTIONAL) - AGENT ANALYSIS
+# ============================================================================
+# Required for `kalshi agent analyze` to use real LLM predictions.
+#
+# Without this, the mock synthesizer returns placeholder "+5% from market" estimates.
+#
+# Backend options: anthropic (default), mock (testing only)
+KALSHI_SYNTHESIZER_BACKEND=anthropic
+#
+# Anthropic API key (get from https://console.anthropic.com/)
+ANTHROPIC_API_KEY=
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The ANTHROPIC_API_KEY key should go before the KALSHI_SYNTHESIZER_BACKEND key

(UnorderedKey)